project-base/storefront/components/Pages/Order/ContactInformation/ContactInformationDeliveryAddress.tsx

```diff
@@ -51,7 +51,6 @@ export const ContactInformationDeliveryAddress: FC = () => {
 
                 setValue(formMeta.fields.deliveryFirstName.name, formValues.firstName, { shouldValidate: true });
                 setValue(formMeta.fields.deliveryLastName.name, formValues.lastName, { shouldValidate: true });
-                setValue(formMeta.fields.deliveryCompanyName.name, formValues.companyName, { shouldValidate: true });
```

Comment:
Why was it removed? I think it would be nice adding motivation for changes to commit messages, as then I would not have to return this back to you to get an answer 🤔

Reply:
Because we don't need (we don't want) this information to be set. It's for the case when user choose pick up place delivery.